test/multistream.test.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+const test = require('tap').test
+const pino = require('..')
+const sink = require('./helper').sink
+
+test('baisc multistream', function (t) {
+  t.plan(2)
+
+  const multistream = pino.multistream()
```

Comment:
I think people are going to get linter errors if this API ships as-is. The `lib/multistream.js` should export a simple factory function that returns an `Object.create()` of an object literal. Or am I missing some technicality that would prevent such?